### PR TITLE
Fix logged out widget menu

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] remove links to tabs in the hamburger menu if logged out
- [x] clean up the file a little bit
- [x] remove wallet flag (its going away this sprint globally)

@keybase/react-hackers 
cc: @songgao

### CODE REVIEW

File: shared/menubar/index.desktop.js [hunk 1]
```diff
@@ -186,12 +182,14 @@ class MenubarRender extends React.Component<Props, State> {
...
-  _menuView(title: string, iconType: Kb.IconType, count: number) {
+  _menuView(title, iconType, count) {
     return (
-      <Kb.Box2 direction="horizontal" style={{width: '100%'}}>
+      <Kb.Box2 direction="horizontal" fullWidth={true}>
         <Kb.Box style={{marginRight: Styles.globalMargins.xsmall, position: 'relative'}}>
           <Kb.Icon type={iconType} color={Styles.globalColors.black_20} fontSize={20} />
-          {!!count && <Kb.Badge badgeNumber={count} badgeStyle={{left: 14, position: 'absolute', top: -2}} />}
+          {!!count && (
+            <Kb.Badge badgeNumber={count || 0} badgeStyle={{left: 14, position: 'absolute', top: -2}} />
```
Comment (buoyad) [inline shared/menubar/index.desktop.js]: `count || 0` necessary?

Comment (chrisnojima) [reply shared/menubar/index.desktop.js]: dunno i just kept what it had before but moved it here instead of all the callers

Comment (buoyad) [reply shared/menubar/index.desktop.js]: ah k I asked because L190 asserts `!!count`

File: shared/menubar/index.desktop.js [hunk 2]
```diff
@@ -200,66 +198,69 @@ class MenubarRender extends React.Component<Props, State> {
...
+    const openApp = startingUp
+      ? []
+      : [{onClick: () => this.props.openApp(), title: 'Open main app'}, 'Divider']
 
-    const openMainApp = [{onClick: () => this.props.openApp(), title: 'Open main app'}]
+    const showFinder =
+      startingUp || loggedOut
+        ? []
+        : [
+            {onClick: () => this.props.showInFinder('/'), title: `Open folders in ${Styles.fileUIName}`},
```
Comment (songgao) [inline shared/menubar/index.desktop.js]: This literally just opens `/`, not `/` under KBFS. I'll fix it in my branch to open /keybase instead.

File: shared/menubar/index.desktop.js (hunk 1 above)
Comment (chrisnojima) [reply shared/menubar/index.desktop.js]: this thing can allow undefined, i didn't look into badge

**Pull request merged**